README.adoc

```diff
@@ -0,0 +1,42 @@
+[[git-plugin]]
+= Tool Environment Plugin
+:toc: macro
+:toc-title:
+
+link:https://ci.jenkins.io/job/Plugins/job/toolenv-plugin/job/master/[image:https://ci.jenkins.io/job/Plugins/job/toolenv-plugin/job/master/badge/icon[Build Status]]
+link:https://github.com/jenkinsci/toolenv-plugin/graphs/contributors[image:https://img.shields.io/github/contributors/jenkinsci/toolenv-plugin.svg?color=blue[Contributors]]
+link:https://github.com/jenkinsci/toolenv-plugin/releases/latest[image:https://img.shields.io/github/release/jenkinsci/toolenv-plugin.svg?label=changelog[GitHub release]]
+link:https://plugins.jenkins.io/toolenv[image:https://img.shields.io/jenkins/plugin/i/toolenv.svg?color=blue&label=installations[Jenkins Plugin Installs]]
+
+[#introduction]
+== Introduction
+
+Tool environment plugin lets you use "tools" in unusual ways, such as from shell scripts.
+
+See http://issues.jenkins-ci.org/browse/JENKINS-4532[JENKINS-4532] for motivation.
+
+Some screenshots are better than a long discussion:
+
+. This is what you define in Hudson's main config screen: +
+[.confluence-embedded-file-wrapper .image-center-wrapper]#image:docs/images/toolenv-01.png[image]#
```

Comment:
Don't need the confluence markers and can help those using screen readers by adding a brief text description of the image
```suggestion
image:docs/images/toolenv-01.png[Jenkins configuration screen]
```